### PR TITLE
Modify load_dataset

### PR DESCRIPTION
**Pull request by paroma**
* Modify load_dataset
* Edit spam notebooks to work without dev and valid set

### CODE REVIEW

File: spam/03_spam_data_slicing_tutorial.py [hunk 1]
```diff
@@ -382,14 +375,14 @@ def textblob_polarity(x):
 
 # For demonstration purposes, we set n_epochs=2
 trainer = Trainer(n_epochs=2, lr=1e-4, progress_bar=True)
-trainer.fit(slice_model, [train_dl_slice, valid_dl_slice])
+trainer.fit(slice_model, [train_dl_slice, test_dl_slice])
```
Comment (bhancock8) [inline spam/03_spam_data_slicing_tutorial.py]: We shouldn't be fitting on test

File: spam/01_spam_tutorial.py [hunk 2]
```diff
@@ -140,20 +140,15 @@
 # %%
 from utils import load_spam_dataset
 
-df_train, df_dev, df_valid, df_test = load_spam_dataset()
+df_train, df_test = load_spam_dataset(load_train_labels=True)
```
Comment (bhancock8) [inline spam/01_spam_tutorial.py]: Why are you loading GT labels for df_train? I don't think we use those or want them here.
